db/migrate/20220401093502_create_physical_server_profiles.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePhysicalServerProfiles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :physical_server_profiles do |t|
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
+      t.string :name
+      t.references :assigned_server, :type => :bigint, :index => true, :references => :physical_server
+      t.references :associated_server, :type => :bigint, :index => true, :references => :physical_server
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
@zitnik schema migrations are held in a separate repository, please create a PR here: https://github.com/ManageIQ/manageiq-schema